src/api/firebase.js

```diff
@@ -153,10 +154,11 @@ export async function updateItem(listId, itemId, checked) {
 	}
 }
 
-export async function deleteItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to delete an existing item. You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+export async function deleteItem(listId, itemId) {
+	const itemRef = doc(db, listId, itemId);
+	try {
+		return await deleteDoc(itemRef);
```

Comment:
This is so great!!! Without the await here you would never hit the catch block even if the function throws/rejects